benchmarks/setup/default/zeebe-values.yaml

```diff
@@ -34,14 +34,14 @@ gateway:
     - name: ZEEBE_GATEWAY_MONITORING_ENABLED
       value: "true"
     - name: ZEEBE_GATEWAY_THREADS_MANAGEMENTTHREADS
-      value: "4"
+      value: "1"
```

Comment:
:question: This means we can only handle one request and/or one response at a time? I know it's async so it's not completely serialized, but it still means you can only do one task at a time? :thinking:

Reply:
For me it looked ok. Be aware that I also increased the replicas.

Reply:
I can't really say. As you said, the benchmark looks fine. It seemed to me like we're removing parallelism here, which would be fine if we could guarantee none of our actor tasks block, but in the past we've seen some of them do, so...

At the end, I just want to avoid people spending time investigating issues which would be due to a very strict resource usage instead of an actual bug. So I would always err on the side of caution and say, OK, if I think the minimum resource usage is X, then let's give it a little more just to be on the safe side and not waste time investigating these kind of issues.